.github/workflows/ci.yml

```
@@ -28,10 +28,10 @@ jobs:
           - '1.10'
           # - 'nightly'
         arch: [x64, x86]
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-latest, windows-latest, macos-13]
         exclude:
-          - os: macos-latest
-            arch: x86
+          - os: macos-13
+          - julia-version: 1.4
```

Needs to be 
```suggestion
            version: 1.4
```
I think.